docs/knowledgebase/getting-started/index.md

```diff
@@ -74,119 +74,69 @@ rustup default stable
 
 Substrate uses [WebAssembly](https://webassembly.org/) (Wasm) to produce portable blockchain
 runtimes. You will need to configure your Rust compiler to use
-[`nightly` builds](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) to allow
-compiled substrate compatible runtimes to Wasm.
+[`nightly` builds](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) to allow you to
+compile Rust code to the Wasm target.
 
 #### Rust Nightly Toolchain
 
-Because the nightly toolchain is a moving target and receives daily changes the chance
-that some of them break the substrate build from time to time is non-negligible.
-
-Therefore it is advised to use a fixed nightly version rather than the latest one to
-build the runtime. You can install a specific version using this command:
+Developers that are building _with_ Substrate (as opposed to the developers building Substrate
+_itself_) should use a specific Rust nightly version that is known to be compatible with the version
+of Substrate they are using. For instance, the Polkadot client specifies this information in its
+[release notes](https://github.com/paritytech/polkadot/releases). The Substrate Developer Hub Node
+Template uses an
+[init script](https://github.com/substrate-developer-hub/substrate-node-template/blob/master/scripts/init.sh)
+and [Makefile](https://github.com/substrate-developer-hub/substrate-node-template/blob/master/Makefile)
+to specify the Rust nightly version and encapsulate the following steps. Use Rustup to install the
+correct nightly:
 
 ```bash
 rustup install nightly-<yyyy-MM-dd>
 ```
 
----
-**NOTE**
-Due to a regression in the rust compiler, using the newest rust nightly for compiling
-the runtime will result in compilation errors. Therefore, it is advised to use the
-following version until this issue is resolved:
-
-```bash
-rustup install nightly-2020-10-06
-```
-
----
-
 #### Wasm Toolchain
 
-Now, configure the choosen nightly version to work with the Wasm compilation target:
+Now, configure the nightly version to work with the Wasm compilation target:
 
 ```bash
 rustup target add wasm32-unknown-unknown --toolchain nightly-<yyyy-MM-dd>
 ```
 
 #### Specifying Nightly Version
 
-When a Substrate based project builds its included runtime it picks the latest
-installed nightly version by default. If the nightly version is incompatible
-you can override that decision by setting the `WASM_BUILD_TOOLCHAIN` environment variable
-when building the project by using the following command:
+Use the `WASM_BUILD_TOOLCHAIN` environment variable to specify the Rust nightly version a Substrate
+project should use for Wasm compilation:
 
-```bash
-WASM_BUILD_TOOLCHAIN=nightly-<yyyy-MM-dd> cargo build
+```sh
```

Comment:
why just change it to `sh` here, but not in the other places?

Reply:
Good call I'll revert it :+1: